src/modules/shortcut_guide/shortcut_guide.vcxproj

```diff
@@ -48,12 +49,12 @@
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
     <LinkIncremental>false</LinkIncremental>
-    <OutDir>$(SolutionDir)$(Platform)\$(Configuration)\modules\</OutDir>
+    <OutDir>$(SolutionDir)$(Platform)\$(Configuration)\modules\$(ProjectName)\</OutDir>

```

Comment:
It probably does not matter, but do we need backslashes at the end, same on few other places ?

Reply:
Probably not, but doesn't hurt to stay there :)